RIM/judges.py

```
@@ -0,0 +1,176 @@
+
+    def _parse_response(self, response: str) -> Dict[str, Any]:
+        try:
+            data = json.loads(response)
+            return {
+                'principles': data.get('principles', []),
+                'score_a': data.get('score_a', 0.0),
+                'score_b': data.get('score_b', 0.0),
+                'explanation': data.get('explanation', ''),
+                'uncertainty': data.get('uncertainty', 0.5)
+            }
+        except:
```

Ruff (0.13.1)

Do not use bare `except`

(E722)